configs/webpack.prod.js

```diff
@@ -64,6 +64,9 @@ module.exports = function(env = {}) {
         test: /\.js$/,
         algorithm: 'gzip',
         asset: '[path].gz[query]'
+      }),
+      new webpack.DefinePlugin({
```

Comment:
Would this be better to pass in via the command line? We pass other args that way